05_syntax_lab/07.py

```diff
@@ -1,8 +1,21 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-
-After each guess print a hint "too large" or "too small" to the user.
-
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
+from random import randint
+
+rand = randint(1,100)
+
+guess = 0
+while True:
+    guess = int(raw_input())
+    if guess == rand:
+        print 'You are right!'
+        break
+    # randomize lie
+    lie = randint(0,2)
+    if lie == 0:

```

Comment:
Python also supports the shorter:
`large = guess < rand if lie == 0 else guess > rand`

Though I'm not sure which is more readable